src/casus/blocks/NotBlock.js

```diff
@@ -0,0 +1,20 @@
+//@flow strict
```

Comment:
Just asking for its own sake. Should we also add a notBlock to the casusBlocks list in the UserModel?

Reply:
Ah, yeah, that's probably a good idea